Header HEADER
string NAME

string NODE_ID
string DATE
string TIME

float64[32] ADC_DATA
